.github/workflows/e2e-matrix.yaml

```diff
@@ -45,6 +66,8 @@ jobs:
       suite: ${{ matrix.suite }}
       git_ref: ${{ needs.resolve-git-ref.outputs.GIT_REF }}
       event_name: ${{ github.event_name }}
```

Comment:
This event_name needs to be passed down through the workflow_call since we're trying to determine if this is triggered by a periodic run, a snapshot run, or a push